libraries/helpers.rb

```diff
@@ -39,9 +39,9 @@ def chef_server_ctl_command(pkg)
       ctl_cmds[pkg]
     end
 
-    def local_provider
-      return Chef::Provider::Package::Dpkg if node['platform_family'] == 'debian'
-      return Chef::Provider::Package::Rpm if node['platform_family'] == 'rhel'
+    def local_package_resource
+      return :dpkg_package if node['platform_family'] == 'debian'
+      return :rpm_package if node['platform_family'] == 'rhel'
```

Comment:
should probably return `:package` here by default, or else raise if you don't want to try to guess that.